packages/rpc-core/test/e2e/state.test.ts

```diff
@@ -46,37 +41,146 @@ describe.skip('e2e state', () => {
       });
   });
 
-  it('retrieves balances', () => {
+  it('getKeys(): retrieves storage keys for ":code"', () => {
     return api.state
-      .getStorage([
-        storage.balances.freeBalance, ALICE
-      ])
-      .then((balance: Balance) => {
-        console.error(balance);
+      .getKeys('0x3a636f6465')
```

Comment:
Even though this is a test, may make sense to put this below the ADDR above (as `cost CODE = '0x3a636f6465'; // :code`)

Reply:
done.